vocabularies/qld-resource-permit-status.ttl

```diff
@@ -0,0 +1,272 @@
+#Version 0.3
+
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix gmps: <http://linked.data.gov.au/def/qld-resource-permit-status/> .
+
+<http://linked.data.gov.au/def/qld-resource-permit-status> a owl:Ontology .
+
+gmps:ConceptScheme a skos:ConceptScheme ;
```

Comment:
change c in first ConceptScheme to lower case. i.e.
gmps:conceptScheme a skos:ConceptScheme